scripts/upload-data.js

```diff
@@ -101,21 +101,171 @@ const main = async () => {
       extension: "geojson",
       field: "geo",
       property: "features",
+      schema: [
+        {
+          name: "geometry",
+          type: "object",
+        },
+        {
+          name: "properties",
+          type: "object",
+          schema: [
+            {
+              name: "cluster_id",
+              type: "number",
+            },
+          ],
+        },
+        {
+          name: "id",
+          type: "number",
+        },
```

Comment:
I see this in the geojson, but do we care that this is there? We can keep this! Though i'm just wondering if we need it as a requirement.

Reply:
I don't know if we care, but it's there and I think is a requirement of geojson itself, so more an extra layer of safety